### PR TITLE
Adding Ras pathway heatmap notebook

### PR DESCRIPTION
**Pull request by gwaygenomics**
Notebook visualizes and outputs Ras Pathway heatmaps for mutation, copy number, and combined summaries.

### CODE REVIEW

File: ras_count_heatmaps.ipynb [hunk 1]
```diff
@@ -0,0 +1,875 @@
...
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.set_style(\"whitegrid\")\n",
+    "sns.heatmap(mut_heatmap_df, linewidths=0.2, linecolor='black', \n",
+    "            cmap='coolwarm', square=True, cbar=True)\n",
```
Comment (dhimmel) [inline ras_count_heatmaps.ipynb]: Diverging colormap to represent a bounded sequential frequency???

File: ras_count_heatmaps.ipynb [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
...HRuGsvVVXJc\nlOTqKvkcQ22O7moBwJO9a/W2ceow1Gwd7iH3Oiav2/ryLypGYpp8fYZbtWpVgoODATh9+jQrV66k\nc+fOz227evVqEhMT2blzJxqNhrNnzzJ69Giio6Oxs7MjISGB2NhYkpOTefjwIc7OknwthBAvEmu5\nNWwqqxk0lZCQQLly5fQu37p1K7t379bdBq5fvz4hISHY2WXtQlRUFK1btyYxMZH9+/fj4+NjkbiF\nEEJYhsZWOlyTxcXF4efnR0pKCnfu3GHNmjW6wU9Pe/jwIYUKFdLl3mZ7+u+IiAg++ugjHj58yMaN\nG6XDFUKIF4yNdLime/qW8h9//MH48eMZOHDgc9sayrW9ceMGd+/e5c0330Sr1TJt2jQePHigq8Ms\nhBCi4NPYFOwO12rSgtzc3ChUqBA2NrlDcnZ2Ji0tjYSEhBzvX7hwgczMTMLDw0lJSaF79+706tUL\nrVarK5ghhBDixaCxtTH5ZczcuXPp06cPvr6+ue60/vnnn/Tt25devXoxffp0Res8j9V0uH///Tf3\n7t1Dq9U+d/mAAQOYO3eubvmZM2cICAggNTWVyMhIgoKCCAsLIywsjOXLl+vqMAshhBCGnDp1imvX\nrrFt2zZmz57Np59+mmP5/PnzGTp0KDt27MDW1pZbt24ZXed5rOIZLkBKSgqBgYEkJiYSFRXF+fPn\nde3WrFnD8OHD+eqrr+jRowfFixfH2dmZlStXEhcXR6FChXKkBzVs2JCEhAT+/PPPHNWphBBCFFzm\neob7/fff07ZtWwCqV69OYmIiSUlJFC1alIyMDM6cOcPixYsBmDFjBgDbt2/Xu44+VlX4wtKk8IUQ\nQqjPXN3KySaNTF630ff66y4EBgby9ttv6zrQfv36MWfOHKpWrUpCQgIDBgygQ4cOnDlzhnr16uHv\n78/06dP1rqNPvlzhGip6sWTJEipVqgTA48eP6dWrF3379gXg6tWrzJ07lwcPHpCRkUG9evWYPHky\nDg4OANy5c4dWrVqxbNky3UEwREmiuJLkdyVt2vTR/2Uf3JZ1En30f4/1tln4fmHVEtuVJP0riddY\ngrySohZK4lVSbEJJcQFjxRCUFG9Qa5+UFBcw1kZJvEpiGTb7nt7la6aVBYxPkK5WUQsl+6RGkRMl\nv1kl8Q6fk2CwzeqpZ
```
Comment (dhimmel) [inline ras_count_heatmaps.ipynb]: Indentation madness

File: ras_count_heatmaps.ipynb (hunk 1 above)
Comment (gwaygenomics) [inline ras_count_heatmaps.ipynb]: by this you mean "coolwarm" would not be what you choose?

File: ras_count_heatmaps.ipynb (hunk 2 above)
Comment (gwaygenomics) [inline ras_count_heatmaps.ipynb]: something I need to fix? it's automatic

**Comment by gwaygenomics** [comment]
thanks for the comments @dhimmel - i've updated the color map and fixed the indentation issue

**Pull request merged**